crates/core/src/operations/create.rs

```diff
@@ -196,6 +198,12 @@ impl CreateBuilder {
         self
     }
 
+    /// Specify whether to raise an error if the table properties in the configuration are not DeltaConfigKeys
+    pub fn with_raise_if_not_exists(mut self, raise_if_key_not_exists: bool) -> Self {
```

Comment:
oops @ion-elgreco I messed this up -- should have been `with_raise_key_if_not_exists` for consistency

Reply:
@vegarsti oops :), you can make a small follow up PR

Reply:
https://github.com/delta-io/delta-rs/pull/2569